packages/template-retail-react-app/jest-setup.js

```diff
@@ -100,6 +100,7 @@ jest.mock('pwa-kit-runtime/utils/ssr-config', () => {
 // TextEncoder is a web API, need to import it
 // from nodejs util in testing environment.
 global.TextEncoder = require('util').TextEncoder
+global.TextDecoder = require('util').TextDecoder
```

Comment:
Duplicate global.

Reply:
I'm curious as to what change was made that required you to add this global? Since it's not in dev, and I don't think there has been much test work up until this point.

Reply:
it was the `node-fetch` change 

> commerce-sdk-isomorphic recently switched from cross-fetch to node-fetch


https://salesforce-internal.slack.com/archives/C0DKK1FJS/p1683662535802599?thread_ts=1683659849.085369&cid=C0DKK1FJS

Reply:
@bendvc I'm not clear on the duplication, I see `global.TextEncoder` / `global.TextDecoder` which are unique